lib/matplotlib/animation.py

```diff
@@ -1785,7 +1785,7 @@ def _draw_frame(self, framedata):
             except TypeError:
                 raise err from None
 
-            # check each item if is artist
+            # check each item if it's artist
```

Comment:
Better just delete this comment.

Reply:
Are you sure, I can also mimic the neighboring comments with `# check if each item is artist`

Reply:
That's basically what the code does; there's no point in having the comment.